### DISCUSSION TITLE
[feature] Add some custom elements to the modal

### DISCUSSION THREAD

**Discussion by Jacky-Young1123** [reown-com/appkit#3697] [Ideas]
### What problem does this new feature solve?

![image](https://github.com/user-attachments/assets/e54d9791-89e3-4110-a218-b4d2127bab83)
As shown in the figure, add some options to the modal for verification after login.

### Describe the solution you'd like

As shown in the figure, add some options to the modal for verification after login.

**Comment by arein** [comment]
@Jacky-Young1123 what should happen if the checkbox is ticked?

Also what exactly will be verified after login?

**Comment by Jacky-Young1123** [comment]
> @Jacky-Young1123如果勾选该复选框会发生什么情况？
> 
> 登录后具体要验证什么？

This is a requirement added by our project. After logging in, the wallet signature verification needs to interact with our own server once. If it is a hardware wallet, signature verification cannot be used and transaction signature authentication is required.
